### PR TITLE
TBRANDS-50: Pull content-api into v2.00

### PR DESCRIPTION
**Pull request by nschubach**
## Description

Update some minor things to bring the content-api into v2.

**This currently fails the test because there are no tests for the transform which will be changing here with resizer v2.  I did not see the purpose of testing this until that update occurs.**

## Jira Ticket

- [TBRANDS-50](https://arcpublishing.atlassian.net/browse/TBRANDS-50)

## Acceptance Criteria

Due to content blocks needing this API, we need to bring forward the content-api content source and make sure it fits in the v2 architecture if needed.

## Test Steps

1. Checkout this branch `git checkout tbrands-50`
2. Run fusion repo with linked blocks `npx fusion start -f -l`
3. See content-api in Pagebuilder local
image

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/content-api-source-block/CHANGELOG.md [hunk 1]
```diff
@@ -1,86 +0,0 @@
-# Change Log
```
Comment (JackHowa) [inline blocks/content-api-source-block/CHANGELOG.md]: was this intentionally deleted? I'm not sure that we'd need to blow this away

**Review by JackHowa — APPROVED** [review]
as long as changelog isn't blown away, this is good to me

File: blocks/content-api-source-block/package.json [hunk 2]
```diff
@@ -2,13 +2,8 @@
   "name": "@wpmedia/content-api-source-block",
   "version": "5.14.1",
   "description": "Content source block for Content API queries",
-  "author": "Joe Grosspietsch <joe.grosspietsch@washpost.com>",
```
Comment (JackHowa) [inline blocks/content-api-source-block/package.json:5]: you don't like beltran and joe?

File: blocks/content-api-source-block/CHANGELOG.md (hunk 1 above)
Comment (nschubach) [reply blocks/content-api-source-block/CHANGELOG.md]: Assuming it's regenerated each build.  The v2 blocks haven't been generating with it, so I've been removing it but I can add it back in.

File: blocks/content-api-source-block/package.json [hunk 3]
```diff
@@ -2,13 +2,8 @@
...
-  "contributors": [
-    "Beltran Caliz <beltran.caliz@washpost.com>"
-  ],
   "homepage": "https://github.com/WPMedia/arc-themes-blocks",
   "license": "CC-BY-NC-ND-4.0",
-  "main": "index.js",
```
Comment (JackHowa) [inline blocks/content-api-source-block/package.json:11]: if main isn't set, this defaults to index.js anyway. we've been using main: ""

**Review by JackHowa — CHANGES_REQUESTED** [review]
minor changes. I don't think the main one is a huge issue but didn't seem intentional to fallback

File: blocks/content-api-source-block/package.json (hunk 2 above)
Comment (nschubach) [reply blocks/content-api-source-block/package.json:5]: Nothing against them.  Just so many hands in this that an author is not really worth including, IMO.  Especially if they leave and we have to go update/clean/remove.

**Review by JackHowa — APPROVED** [review]
entryfile looks good

**Pull request merged**